uninstall.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Fired when the plugin is uninstalled.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ */
+
+// If uninstall not called from WordPress, then exit.
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	exit;
+}
+
+/**
+ * Deletes all poems from the database when the plugin is uninstalled.
```

Comment:
I'd highly advise NOT deleting user's content as users will not be happy! 

Only delete the demo poem that the plugin added if it exists.

Reply:
You could have an admin screen with an option to "Delete ALL poems on plugin uninstall." If this option is enabled at the point of uninstalling, go ahead with the full removal—that way, you've got positive buy-in from the user.

The use case here is someone trying multiple plugins to manage their poems rather than wanting the underlying content to be deleted as they try the plugins!

Reply:
I like this idea, I will add a new issue